src/repm-invoker.ts

```diff
@@ -133,6 +133,7 @@ export type CommitTransactionResponse = {
 
 type BeginSyncRequest = {
   batchPushURL: string;
+  clientViewURL: string;
```

Comment:
@phritz I assume this is where it is handed in?

Reply:
Yes, precisely.